src/recordlinker/routes/person_router.py

```diff
@@ -75,3 +100,34 @@ def update_person(
 
     person = service.update_person_cluster(session, patients, person, commit=False)
     return schemas.PersonRef(person_reference_id=person.reference_id)
+
+
+@router.post(
+    "/{person_reference_id}/merge",
+    summary="Merge Person clusters",
+    status_code=fastapi.status.HTTP_200_OK,
+)
+def merge_person_clusters(
+    person_reference_id: uuid.UUID,
+    data: typing.Annotated[schemas.PersonRefs, fastapi.Body()],
+    session: orm.Session = fastapi.Depends(get_session),
+) -> schemas.PersonRef:
+    """
+    Merges Person cluster(s) into the Person cluster referenced by `person_reference_id`.
+    """
+    # Get the person_reference_id the person clusters will be merged into
+    per = service.get_person_by_reference_id(session, person_reference_id)
+
+    if per is None:
+        raise fastapi.HTTPException(status_code=fastapi.status.HTTP_404_NOT_FOUND)
+
+    # Get all persons by person_reference_id that will be merged
```

Comment:
Nit: since we're using person_reference_id as a more general construct here to refer to "an ID that uniquely references a given person cluster", what would you say to changing the parameter name from `person_reference_id` to something like `merge_into_id`? Right now, person_reference_id is getting a little overloaded to me.

Reply:
Yes, I like that suggestion.